### PR TITLE
fixed keyboardVisible not updating

### PR DESCRIPTION
**Pull request by likewei92**
keyboardVisible wasn't updating properly

### CODE REVIEW

File: src/input.service.ts [hunk 1]
```diff
@@ -268,7 +267,10 @@ export class InputService {
    * Inputpane and boolean to indicate whether it's visible
    */
   private inputPane = (<any>window).Windows ? Windows.UI.ViewManagement.InputPane.getForCurrentView() : null;
-  public keyboardVisible = new BehaviorSubject(false);
+
+  public get keyboardVisible(): boolean {
+    return (this.inputPane.occludedRect.y !== 0);
```
Comment (connor4312) [inline src/input.service.ts]: Unnecessary parens, also you probably want `this.inputPane && this.inputPane.occludedRect.y !== 0`

**Comment by connor4312** [comment]
Hey, please fix the thing I commented on, that will cause the site to not work on desktop.

**Comment by SimonSchick** [comment]
@likewei92 You shouldn't have merged that as you have not addressed connors notes, my approval was depending on you fixing connor's notes 😐

**Comment by likewei92** [comment]
Oops accidentally pushed to master. I fixed the change though

**Pull request merged**